docs/_basic/listening_actions.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Listening to actions
+slug: action-listening
+order: 6
+---
+
+<div class="section-content">
+Your app can listen to user actions like button clicks, menu selects, and message actions using the `action` method.
+
+Actions can be filtered on  an `action_id` of type string or RegExp. `action_id`s act as unique identifiers for interactive components on the Slack platform. 
+
+You’ll notice in all `action()` examples, `ack()` is used. It is required to call the `ack()` function within an action listener to acknowledge the event was received from Slack. This is discussed in the [acknowledging actions section](#action-acknowledging).
+
+</div>
+
+```javascript
+// Your middleware will be called every time an interactive component with the action_id "approve_button" is triggered
+app.action('approve_button', async ({ ack, say }) => {
+  ack();
+  // Update the message to reflect the action
+});
+```
+
+<details class="secondary-wrapper">
+<summary class="section-head" markdown="0">
+<h4 class="section-head">Listening to actions using a constraint object</h4>
+</summary>
+
+<div class="secondary-content" markdown="0">
+You can use a constraints object to listen to `callback_id`s, `block_id`s, and `action_id`s (or any combination of them). Constraints in the object can be of type string or RegExp.
+</div>
+
+```javascript
+// Your middleware will only be called when the action_id matches 'select_user' AND the block_id matches 'assign_ticket'
+app.action({ action_id: 'select_user', block_id: 'assign_ticket' },
+  async ({ action, ack, context }) => {
+    ack();
+    try {
+      const result = await app.client.reactions.add({
+        token: context.botToken,
+        name: 'white_check_mark',
+        timestamp: action.action_ts,
```

Comment:
`action_ts` represents when the user selected an item in the menu, not the `ts` of a message. the `timestamp` argument needs the `ts` of a message.